automation/BrowserManager.py

```diff
@@ -266,7 +267,8 @@ def close_browser_manager(self):
             return
 
         # Send the shutdown command
-        self.command_queue.put(("SHUTDOWN",))
+        command = Shutdown_Command()
+        self.command_queue.put((command,))
```

Comment:
```suggestion
        self.command_queue.put(command)
```
Because currently the shutdown code doesn't run shutdown code as `type((command,))` is `tuple` and not `Shutdown_Command` as expected